SU-SWS/ood_giving_site
# Initial changes with some new components

# READY FOR REVIEW

# Summary
- Various components and scss files. Merged in @katrialesser's components and changes also.

# Needed By (Date)
- Sooner the better

# Urgency
- N/A

# Steps to Test

1. Pull this branch
1. `nvm use`
1. `gatsby develop`
1. Check that things look as expected in Storyblok ADAPT Digital space

# Affected Projects or Products
- Does this PR impact any particular projects, products, or modules?

# Associated Issues and/or People
- JIRA ticket 
https://stanfordits.atlassian.net/browse/ADAPT-558?atlOrigin=eyJpIjoiY2I5ZTkyMDNmOWI5NGFlNDk1NDYwNjZmZDZlNTViNmUiLCJwIjoiaiJ9
